main.go

```diff
@@ -137,7 +147,8 @@ func parseFlags() *config {
 	flag.StringVar(&cfg.telemetryPath, "web.telemetry-path", "/metrics", "Address to listen on for web endpoints.")
 	flag.StringVar(&cfg.logLevel, "log.level", "debug", "The log level to use [ \"error\", \"warn\", \"info\", \"debug\" ].")
 	flag.BoolVar(&cfg.readOnly, "read.only", false, "Read-only mode. Don't write to database.")
-
+	flag.IntVar(&cfg.haGroupLockId, "ha.group-advisory-lock-id", 0, "Unique advisory lock id per adapter high-availability group. Set it if you want to use leader election implementation based on PostgreSQL advisory lock")
+	flag.BoolVar(&cfg.restElection, "rest.election", false, "Enable REST interface for the leader election")
```

Comment:
I would group this parameter with the above one under a common prefix, e.g., `leader-election.http` and `leader-election.advisory-lock` or something. Then it is clear that these are parameter related to similar functionality.

Also, just wondering if it should be possible to have both enabled simultaneously? If not, I could imagine having one string parameter that allows `leader-election=http`, `leader-election=lock`, leader-election=off` or similar.